pkg/cli/cluster_free_disk_space.go

```diff
@@ -0,0 +1,133 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	rookcli "github.com/rook/rook/pkg/client/clientset/versioned"
+
+	clusterspace "github.com/replicatedhq/kurl/pkg/cluster/space"
+)
+
+const (
+	// defaultOpenEBSPodImage is the image used during disk free check for openebs storage.
+	// this is the same image used by the pvmigrate project, it may be any image containing
+	// 'df' and 'cat' commands.
+	defaultOpenEBSPodImage          = "eeacms/rsync:2.3"
```

Comment:
How we will keep it maintained?
Will we need to update the default value when pvmigrate change it there?
Can we just use it from there or at least add a TODO for we export this const there and get here?
Is that something that make sense?

Reply:
We don't need to keep them both in sync, no. There is a flag that can be used to replace the image as well. What else to you think is necessary ? If you prefer I can simply remove the default and let then customise in the installer.

Reply:
I think the default value is great and it probably is the latest release one.
Yes, i see that we can use the flag but would be great if we do not need update that manually when a new release for example happens. However, that is not a blocker for we get this one merged.

Reply:
As this is not a blocker I am going to resolve it.